.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Build
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  Build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [2.7, 3.6]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install google-compute-engine
+          pip install twine
+          pip install -r requirements-dev.txt
+          python setup.py install
+      - name: Run Tests
+        run: |
+          nosetests --with-coverage --cover-xml tests/
+      - name: Upload coverage report to CodeCov
+        uses: codecov/codecov-action@v1
+        with:
+          file: coverage.xml # optional
+          fail_ci_if_error: true # optional (default = false)
```

Comment:
Will this send a Slack notification if any tests fail?

Reply:
I had actually forgotten to put the slack config in so thanks for catching that. 

This will notify us in slack anytime the master build fails (due to failed tests, failed coverage upload, or anything else)